### ISSUE TITLE
Is there a way to find results with context lines before and after matched lines?

### ISSUE THREAD

**Issue by pansila** [nushell/nushell#9555]
### Question

Just like grep's '-A' and '-B' flags, otherwise I have to str join rows and search them with grep since most data in nushell is records in a table, thanks in advance.

**Comment by fdncred** [comment]
no, this functionality doesn't exist in the `find` command. It may be interesting to have something like this but I think it may be a little complicated to add.

**Comment by WindSoilder** [comment]
Closing the issue because the question is answered.
